### PR TITLE
Fixes #4, #6, #3, #1

### PR DESCRIPTION
**Pull request by frogcrush**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # ➤ bchydro-outages
 
-This is an *unofficial* repository that provides a Git history of outages at [BC Hydro](https://bchydro.com), a Canadian utility.
+This is an *unofficial* repository that provides a Git history of outages at [BC Hydro](https://bchydro.com), a Canadian utility company.
```
Comment (marcolussetti) [inline README.md]: I think we can leave the original language.

File: .github/workflows/scrape.yml [hunk 2]
```diff
@@ -12,21 +12,14 @@ jobs:
...
+          curl https://www.bchydro.com/power-outages/app/outages-map-data.json | jq . > bchydro-outages.json
       - name: Add results to config
         run: |-
           git config user.name "Automated"
           git config user.email "actions@users.noreply.github.com"
           git add -A *.json
           timestamp=$(date -u -Is)
           git commit -m "Update data on ${timestamp}" || exit 0
-          git push
+          git push
```
Comment (marcolussetti) [inline .github/workflows/scrape.yml]: Any chance we can have the EOF char back?

**Review by marcolussetti — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (frogcrush) [reply README.md]: A utility would be "gas" or "electricity" as a purchasable thing, whereas a utility company would be "BC Hydro" or "FortisBC" no?

**Review by marcolussetti — COMMENTED** [review]
Works for me.

**Pull request merged**